src/ansys/mechanical/core/embedding/runtime.py

```diff
@@ -44,6 +44,18 @@ def __register_function_codec():
     Ansys.Mechanical.CPython.Codecs.FunctionCodec.Register()
 
 
+def _bind_assembly_for_explicit_interface(assembly_name: str):
+    """Bind the assembly for explicit interface implementation."""
+    import clr
+
+    assembly = clr.AddReference(assembly_name)
+    from Python.Runtime import BindingManager, BindingOptions
```

Comment:
@dipinknair This will throw an error if `pythonnet` isued rather than `ansys-pythonnet`. Up to now that isn't an error, it is just a warning.

This change will turn it into an error. I don't think that's intentional. You can check to see if `pythonnet` is used before doing this.